04-spatial-operations.Rmd

```diff
@@ -873,7 +873,7 @@ clip = rast(xmin = 0.9, xmax = 1.8, ymin = -0.45, ymax = 0.45,
             resolution = 0.3, vals = rep(1, 9))
 elev[clip]
 # we can also use extract
-# terra::extract(elev, ext(clip))
+terra::extract(elev, ext(clip))
```

Comment:
This was deliberate, to reduce the amount of output. Thoughts @Nowosad ? We could create a new unevaluated chunk.

Reply:
Yes -- this is deliberate -- to show the alternative code without running it.